docs/src/installation.md

```diff
@@ -37,11 +37,11 @@ Solver support in Julia is currently provided by writing a solver-specific packa
 | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- | ------------------------- | ----------- | ----- | ----- | ----- | ----- | ----- | ----- |
 | [Artelys Knitro](http://artelys.com/en/optimization-tools/knitro)                | [KNITRO.jl](https://github.com/JuliaOpt/KNITRO.jl)                              | `KnitroSolver()`          |  Comm.      |       |       |       | X     | X     |       |
 | [BARON](http://archimedes.cheme.cmu.edu/?q=baron>)                               | [BARON.jl](https://github.com/joehuchette/BARON.jl)                             | `BaronSolver()`           |  Comm.      |       |       |       | X     | X     |       |
-| [Bonmin](https://projects.coin-or.org/Bonmin)                                    | [AmplNLWriter.jl](https://github.com/JackDunnNZ/AmplNLWriter.jl)                | `BonminNLSolver()` *      |  EPL        | X     |       | X     | X     | X     |       |
+| [Bonmin](https://projects.coin-or.org/Bonmin)                                    | [AmplNLWriter.jl](https://github.com/JuliaOpt/AmplNLWriter.jl)                | `AmplNLWriter(CoinOptServices.bonmin)` *      |  EPL        | X     |       | X     | X     | X     |       |
```

Comment:
Double check that the table complies with this alignment

Reply:
![image](https://user-images.githubusercontent.com/8177701/33300002-014d5da2-d453-11e7-8836-e403b9328fa3.png)